pkg/diagnostics/diagnostics.go

```diff
@@ -78,6 +89,8 @@ func RegisterClientInfoSource(
 		clientInfo := map[string]interface{}{
 			"network_id":       clientID,
 			"ethereum_address": key.NetworkPubKeyToEthAddress(clientPublicKey),
+			"network_addrs":    connectionManager.NetAddrStrings(),
+			"datetime":         time.Now().Format("2006-01-02 15:04:05"),
```

Comment:
Circling back here after a long time, I realize---what's the reasoning for including a timestamp in the JSON here? I believe Go's default HTTP response includes a `Date` header that should make this redundant.

Reply:
Thank you for your reminder, I mistakenly thought that Diagnostics also has a tick like Metrics.

But the timestamp in JSON may be more convenient than the HTTP header, and there is only a little overhead. I can also delete it if you want to delete it.

Is there any other problem with the code, let me update code together